src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -38,7 +38,8 @@ public enum IamAction {
   UPDATE_BILLING_ACCOUNT,
   LINK,
   // journal
-  VIEW_JOURNAL;
+  VIEW_JOURNAL,
+  VIEW_SNAPSHOT_BUILDER_CONFIG;
```

Comment:
Relevant Sam PR: https://github.com/broadinstitute/sam/pull/1161

Reply:
If you're changing naming from "Snapshot Builder Config" to "Snapshot Builder Settings", would it make sense to also change the name of the Sam action accordingly?

Reply:
I see that you made this change: https://github.com/broadinstitute/sam/pull/1161/commits/6b917ca9ad4c31681e13a6e34bac06c43c04495b